tests/unit_tests/arelle/test_pluginmanager.py

```diff
@@ -89,14 +89,14 @@ def test_plugin_manager_reset():
             # Test data
             ("tests/unit_tests/arelle", "functionsMaths", "xyz"),
             # Expected result
-            ("functionsMaths", "tests/unit_tests", "xyz")
+            ("functionsMaths", f"tests{os.sep}unit_tests", "xyz")
```

Comment:
nit: I think using [os.path.normcase()](https://docs.python.org/3/library/os.path.html?highlight=path#os.path.normcase) to consume these paths in the test body would be a little more resilient (filesystem capitalization support, etc) to any potential future changes.

Reply:
You mean wrap the whole string of the exected result in that function?

Reply:
Yes. I don't feel strongly though. I also think the `test_data` paths should be normalized to correctly exercise the code under test on windows.

Reply:
OK, I'll leave it to be then. `os.sep` is used alot in Arelle so maybe an issue should be raised to replace all of them at once?

Reply:
@stevenbronson-wk sorry, I don't quite understand what you mean here. Where should that go?